checkmates/pipelines/utils.py

```diff
@@ -65,6 +69,9 @@ def _make_component_list_from_actions(actions):
     if indices_to_drop:
         indices_to_drop = sorted(set(indices_to_drop))
         components.append(DropRowsTransformer(indices_to_drop=indices_to_drop))
+    if cols_to_normalize:
+        cols_to_normalize = sorted(set(cols_to_normalize))
```

Comment:
do we need to sort the columns?

Reply:
yeah they don't actually need to be sorted, removed this